iosApp/CICDTests.xctestplan

```diff
@@ -0,0 +1,43 @@
+{
+  "configurations" : [
+    {
+      "id" : "907BC089-7FF9-4CC2-B76C-526FB9AF9732",
+      "name" : "Configuration 1",
+      "options" : {
+        "uiTestingScreenshotsLifetime" : "keepNever"
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "targetForVariableExpansion" : {
+      "containerPath" : "container:Survey.xcodeproj",
+      "identifier" : "3AC192947DB2FAD63AFCCAEB",
+      "name" : "Survey"
+    }
+  },
+  "testTargets" : [
+    {
+      "target" : {
+        "containerPath" : "container:Survey.xcodeproj",
+        "identifier" : "0DC8F1540991EF674B7401A8",
+        "name" : "SurveyTests"
+      }
+    },
+    {
+      "skippedTests" : [
```

Comment:
So you skip these tests on the local?

Reply:
On local we will run another Scheme that contains all the tests 🙏

Reply:
@blyscuit I don't see you run the scheme with this test plan anywhere: https://github.com/blyscuit/kmm-swiftUI-IC/pull/97/files#diff-875174c2ce4bb9989253a0dcc3c02e43a7ed434ef8c9583846f2f5fbcaf9394eR90

Reply:
This test plan is run from `Survey Staging` which is the default testing scheme on GH Actions. https://github.com/blyscuit/kmm-swiftUI-IC/pull/97/files#diff-89b0b339ece8864adc397142cc5ccc5247653a906fe26d2d06c9fb058d7a7533R31